src/api/userInformation.ts

```diff
@@ -13,12 +13,14 @@ const getInformation = (user: any) =>{
 
 const sendUserInformation = async (user: any) => {
     const information = getInformation(user);
-    try{
-        await api.post('/users', information)
-    }catch(err) {
-        console.log(err);
-    }
-    
+    return (api.post('/users', information)
+    .then((response)=> {
```

Comment:
dlaczego? wroc do poprzedniej formy

Reply:
Nie lepiej byłoby dorzucić jakiś interceptor, żeby obsługiwać te wyjątki? nie trzeba by było chyba wtedy tych try/catch wrzucac do każdego requesta

Reply:
A co do tej zmiany requesta to ja Michałowi powiedziałem, żeby zmienił, bo widziałem, że miał użyte await i then razem w sensie coś w stylu await api.post('/users', information).then(...   a nie powinno się z tego co słyszałem await i then razem stosować, więc kazałem mu await wywalić, ale teraz widzę, że wrzucone było z samym await więc było git.

Reply:
Dobra, cofne jak było

Reply:
Zmieniłem na then, gdyż teraz Backend zwraca id usera i musze jakoś pobrać te dane i przenieść do store usera. Ale teraz zrobię inaczej. Zostanie try i catch ale dodam returna z niego i powinno być okej.

Reply:
a nie moze byc cos takiego?
const result = await api.post('/users', information)
return result.data
ewentualnie jak ma byc te try catch to
let result = null
try {
result = await api.post('/users', information)
return result
} catch (e) {
}
albo tak, bo nwm jak wy ale nie przepadam za "letami"
try {
 return (await api.post('/users', information)).data
} catch (e) {
}

Reply:
Sory nie wiedziałem, że pisałeś już jakoś wrzuciłem, chyba nawet podobnie jak chodziło ci w ostatnim przykładzie. Zrób review na nowo a tą sekcje daje jako resolve.